<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="headerText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Mouse over an option to begin.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewProcessesButton">
        <property name="text">
         <string>View Processes</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="8" column="3">
       <widget class="QCheckBox" name="checkBox32">
        <property name="text">
         <string>Blocked</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QCheckBox" name="checkBox35">
        <property name="text">
         <string>Wchan</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkBox25">
        <property name="text">
         <string>RSSLim</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QCheckBox" name="checkBox27">
        <property name="text">
         <string>Endcode</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="checkBox37">
        <property name="text">
         <string>CNswap</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QCheckBox" name="checkBox50">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Env Start</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="checkBox33">
        <property name="text">
         <string>Sigignore</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="checkBox8">
        <property name="text">
         <string>TPGID</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QCheckBox" name="checkBox24">
        <property name="text">
         <string>RSS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox11">
        <property name="text">
         <string>CMinflt</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBox22">
        <property name="text">
         <string>Starttime</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="checkBox20">
        <property name="text">
         <string>Numthreads</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QCheckBox" name="checkBox44">
        <property name="text">
         <string>C Guest Time</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QCheckBox" name="checkBox49">
        <property name="text">
         <string>Arg End</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="checkBox12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Majflt</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QCheckBox" name="checkBox43">
        <property name="text">
         <string>Guest Time</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkBox21">
        <property name="text">
         <string>ITrealvalue</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="checkBox29">
        <property name="text">
         <string>Kstkesp</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QCheckBox" name="checkBox40">
        <property name="text">
         <string>RT_priority</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QCheckBox" name="checkBox51">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Env End</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox3">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="checkBox19">
        <property name="text">
         <string>Nice</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox15">
        <property name="text">
         <string>Stime</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox10">
        <property name="text">
         <string>Minflt</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="checkBox30">
        <property name="text">
         <string>Kstkeip</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QCheckBox" name="checkBox39">
        <property name="text">
         <string>Processor</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="checkBox26">
        <property name="text">
         <string>StartCode</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QCheckBox" name="checkBox45">
        <property name="text">
         <string>Start Data</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox4">
        <property name="text">
         <string>Parent PID</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="checkBox38">
        <property name="text">
         <string>Exit Signal</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="checkBox23">
        <property name="text">
         <string>VSize</string>
        </property>
       </widget>
      </item>
      <item row="13" column="3">
       <widget class="QCheckBox" name="checkBox52">
        <property name="text">
         <string>Exit Code</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox6">
        <property name="text">
         <string>Session</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox1">
        <property name="text">
         <string>PID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>COMM</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QCheckBox" name="checkBox31">
        <property name="text">
         <string>Signal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TTY_NR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox5">
        <property name="text">
         <string>PGRP</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QCheckBox" name="checkBox46">
        <property name="text">
         <string>End Data</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox14">
        <property name="text">
         <string>Utime</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="checkBox41">
        <property name="text">
         <string>Policy</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox17">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>CStime</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox18">
        <property name="text">
         <string>Priority</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="checkBox34">
        <property name="text">
         <string>Sigcatch</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QCheckBox" name="checkBox47">
        <property name="text">
         <string>Start BK</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QCheckBox" name="checkBox42">
        <property name="text">
         <string>Aggregated Block</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QCheckBox" name="checkBox48">
        <property name="text">
         <string>Arg Start</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox13">
        <property name="text">
         <string>CMajflt</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QCheckBox" name="checkBox28">
        <property name="text">
         <string>Startstack</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox9">
        <property name="text">
         <string>Flags</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QCheckBox" name="checkBox36">
        <property name="text">
         <string>Nswap</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="checkBox16">
        <property name="text">
         <string>Cutime</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="selectDeselectAll">
        <property name="text">
         <string>Select / Deselect All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGazelle">
    <property name="title">
     <string>Gazelle</string>
    </property>
   </widget>
   <addaction name="menuGazelle"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
